stable/weave-cloud/ci/e2e-values.yaml

```diff
@@ -0,0 +1,13 @@
+# CI values for weave-cloud.
+
+token: demo
```

Comment:
The only thing that changes for CI seems to be the token. Don't duplicate the rest.